bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -235,6 +253,8 @@ service PrivateSettlementService {
   rpc WithdrawAllBases(WithdrawAllBasesRequest) returns (WithdrawAllBasesResponse);
   // Endpoint to get the information about the network the oracle is running on
   rpc Info(InfoRequest) returns (InfoResponse);
+  // Endpoint to set details on how the quotes should be withdrawn.
+  rpc SetQuotesWithdrawalDetails(SetQuotesWithdrawalDetailsRequest) returns (SetQuotesWithdrawalDetailsResponse);
```

Comment:
Is it really necessary? Currently the contracts dont support such settings, so this would have to be a grpc service level feature. 
And currently the grpc services are basically smart wrappers around the contracts with no native logic or storage env setup.
As we are adding the `optional string deposit_address = 2;` to the relevant endpoints, wouldn't that be enough to know whether to send the quotes to a different address?

Reply:
It's not necessary, it's helpful - MM could call it once on config and don't bother keeping this detail internally. But if that is an issue, MM can just keep this knowledge itself and pass with every withdrawal.